### PR TITLE
Update wufb-compliancedeadlines.md

### PR DESCRIPTION
**Pull request by vivianxiang123**
## Why

- Closes #[Issue Number]

### CODE REVIEW

**Comment by vivianxiang123** [comment]
@mestew  I added one sentence to clarify commercial deadline policy. Please have a look

File: windows/deployment/update/wufb-compliancedeadlines.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The deadline calculation for both quality and feature updates is based off the t
 The grace period for both quality and feature updates starts its countdown from the time of a pending restart after the installation is complete. As soon as installation is complete and the device reaches pending restart, users are able to schedule restarts during the grace period and Windows can still automatically restart outside of active hours if users choose not to schedule restarts. Once the *effective deadline* is reached, the device tries to restart during active hours. (The effective deadline is whichever is the later of the restart pending date plus the specified deadline or the restart pending date plus the grace period.) Grace periods are useful for users who may be coming back from vacation, or other extended time away from their device, to ensure a forced reboot doesn't occur immediately after they return.
 
 > [!NOTE]
+> When **Specify deadlines for automatic updates and restarts** is used, update will be downloaded and installed as soon as they are offered.
```
Comment (mestew) [inline windows/deployment/update/wufb-compliancedeadlines.md:49]:
```suggestion
> - When **Specify deadlines for automatic updates and restarts** is used, updates will be downloaded and installed as soon as they are offered. 
```

File: windows/deployment/update/wufb-compliancedeadlines.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ The deadline calculation for both quality and feature updates is based off the t
...
 
 > [!NOTE]
+> - When **Specify deadlines for automatic updates and restarts** is used, updates will be downloaded and installed as soon as they are offered. 
 > When **Specify deadlines for automatic updates and restarts** is used, download, installation, and reboot settings stemming from the [Configure Automatic Updates](waas-restart.md#schedule-update-installation) are ignored.
```
Comment (mestew) [inline windows/deployment/update/wufb-compliancedeadlines.md:50]:
```suggestion
> - When **Specify deadlines for automatic updates and restarts** is used, download, installation, and reboot settings stemming from the [Configure Automatic Updates](waas-restart.md#schedule-update-installation) are ignored.
```

**Comment by Lo236816** [comment]
Revert documentation changes

**Pull request merged**